homeassistant/components/roon/media_player.py

```diff
@@ -533,17 +533,17 @@ def join(self, join_ids):
             if name is None:
                 _LOGGER.error("No roon player found for %s", entity_id)
                 return
-            if name not in sync_available.keys():
+            if name not in sync_available:
```

Comment:
There are five more `.keys()` that we can remove.

Reply:
Oops...